include/glfw/window/error.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdexcept>
+
+namespace wnd
+{
+
+class Error : public std::runtime_error
+{
+
+  private:
+    int m_error_code;
+
+  public:
+    // We can use is it for our purposes because glfw doesn't throw 0 error code.
+    static constexpr int k_user_error = 0;
+
+    Error( int error_code, std::string description )
```

Comment:
Description should be const std::string&

Reply:
Agree.